app/mailers/concerns/addresses.rb

```diff
@@ -2,7 +2,7 @@ module Addresses
   include ActiveSupport::Concern
 
   def prison_mailbox_email
-    Rails.configuration.prison_data[@visit.prisoner.prison_name]['email']
+    Prison.find(@visit.prisoner.prison_name).email
```

Comment:
couldn't this just be `@visit.prisoner.prison.email`? It would be even nicer if we could delegate through: `@visit.prison.email` or `@visit.prison_email` perhaps.